voltdb/assets/configuration/spec.yaml

```diff
@@ -26,6 +26,46 @@ files:
               type: boolean
               example: false
 
+          - name: statistics_components
+            description: |
+              See: https://docs.voltdb.com/UsingVoltDB/sysprocstatistics.php
+              The VoltDB integration does not collect metrics from all components by default, some are redundant, and
+              some can have a very high cardinality associated.
+              The following components are supported:
+               - COMMANDLOG
+               - CPU
+               - EXPORT
+               - GC
+               - IDLETIME
+               - IMPORT
+               - INDEX
+               - IOSTATS
+               - LATENCY
+               - MEMORY
+               - PROCEDURE
+               - PROCEDUREOUTPUT
+               - PROCEDUREPROFILE
+               - QUEUE
+               - SNAPSHOTSTATUS
+               - TABLE
+            display_priority: 1
+            required: true
```

Comment:
is it? There is a default https://github.com/DataDog/integrations-core/pull/9233/files#diff-f939c7893509b57adc4ec16fafbd00b10792528fb4c68c1a5b3f89e961184f0aR54

Reply:
updated